padevents/padevents.py

```diff
@@ -437,14 +436,12 @@ async def aepset(self, ctx, key, k, v):
     async def aepget(self, ctx, key):
         pingroles = await self.config.guild(ctx.guild).pingroles()
         if key not in pingroles:
-            await ctx.send("That key does not exist.")
-            return
+            raise ClientInlineTextException("That key does not exist.")
```

Comment:
Can you say what the key was & print the list of all keys

The key `{key}` does not exist. The following keys are defined:

```
list
of
keys
```

You can use tsutils cancellation message if this isn't possible with the error message